examples/hosts/app-integration/external-data/tests/externalDataService.test.ts

```diff
@@ -55,8 +56,8 @@ describe("mock-external-data-service", () => {
 		await closeServer(_externalDataService);
 	});
 
-	async function getCurrentExternalData(): Promise<TaskData> {
-		const fetchResponse = await externalDataSource!.fetchData();
+	async function getCurrentExternalData(): Promise<ITaskData> {
+		const fetchResponse = await externalDataSource!.fetchData("task-list-1");
```

Comment:
```suggestion
		const fetchResponse = await externalDataSource!.fetchData(externalTaskListId);
```
Let's give that constant more work 😄 .